src/style.css

```
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars * {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star {
+  margin-right: 4px;
+
+  width: 16px;
+  height: 16px;
+}
+
+.stars--1 > :first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 > :first-child,
+.stars--2 > :nth-child(2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 > :first-child,
+.stars--3 > :nth-child(2),
+.stars--3 > :nth-child(3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 > :first-child,
+.stars--4 > :nth-child(2),
+.stars--4 > :nth-child(3),
+.stars--4 > :nth-child(4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 > :first-child,
+.stars--5 > :nth-child(2),
+.stars--5 > :nth-child(3),
+.stars--5 > :nth-child(4),
+.stars--5 > :nth-child(5) {
+  background-image: url(images/star-active.svg);
+}
```

```suggestion
  .stars--1 :nth-child(1),
	.stars--2 :nth-child(-n+2),
	.stars--2 :nth-child(-n+3),
	.stars--2 :nth-child(-n+4),
	.stars--2 :nth-child(-n+5), {
	  background-image: url(./images/star-active.svg);
	}
```
Можна сильно скоротити код, використавши nth-child таким чином

---

Wow, дякую!